components/brave_wallet/browser/brave_wallet_utils_unittest.cc

```diff
@@ -273,4 +273,284 @@ TEST(BraveWalletUtilsUnitTest, Mnemonic) {
   }
 }
 
+TEST(BraveWalletUtilsUnitTest, EncodeString) {
+  std::string output;
+  EncodeString("one", &output);
```

Comment:
ASSERT_TRUE here and for other `EncodeString` calls

Reply:
This and below comments are addressed in https://github.com/brave/brave-core/pull/8437/commits/2ff5dfe688cf52584c76ed97398246637ea2d0fc.